src/components/Jobs/RunningJobs/JobDetails/JobDetails.js

```diff
@@ -42,14 +42,14 @@ const JobDetails = props => {
         <span data-test-running-job-file-name>{get(exportedFiles, '0.fileName')}</span>
       </div>
       <div className={css.delimiter}>
-        <span data-test-running-job-hr-id>{get(job, 'hrId', '')}</span>
+        <span data-test-running-job-hr-id>{hrId}</span>
         {runBy && (
-          <span data-test-running-job-triggered-by>
-            <FormattedMessage
-              id="ui-data-export.triggeredBy"
-              values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
-            />
-          </span>
+        <span data-test-running-job-triggered-by>
```

Comment:
Indentation is needed:)

Reply:
Before 
{runBy ... ?

My esLint not show error on this, sorry

Reply:
I pushed, lint did not seem to bother.